Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -133,6 +133,7 @@ export function install(args: Arguments) : Result {
                         ]),
                     ],
                 },
+                requireCbDependencies: true,

```

Comment:
>requireCbDependencies [](start = 16, length = 21)

We should move this into the `unsafe`, and change the name to not include CB, e.g., `useGlobalDependencies` (which should pull `unsafe_GlobalPassthroughEnvVars` and `unsafe_GlobalUntrackedScopes` into the pip).